frontend/mr-admin-flate/src/utils/Utils.ts

```diff
@@ -19,3 +19,15 @@ export function formaterDato(dato?: string | Date, fallback = ""): string {
 
   return result;
 }
+
+const zeroPad = (value: number): string => {
+  return value < 10 ? `0${value}` : value.toString();
+};
+
+export const datoStrengUtenTid = (date?: Date): string => {
```

Comment:
Kunne evt. vurderert å introdusert et bibliotek for dette, f.eks. [date-fns](https://date-fns.org/v2.29.3/docs/formatISO#examples): `formatISO(date, { representation: 'date' })`

Reply:
Ja, kunne det, men samtidig er det jo enkelt å gjøre det selv 😄